### PR TITLE
Engagement/jessica/14422 idaho migration pre work

### PR DESCRIPTION
**Pull request by JessicaWNava**
This PR implements the pre migration work for Idaho. It is based off of Indiana because it needs the changes to the simple report transform, but will be based off master once that PR is merged.

Test Steps:
1. Review that the changes in organizations.yml match ID previous settings
2. Check that the resulting message matches the outbound message from the Covid pipeline
3. Make sure the tests pass

## Changes
- Added settings for ID to organizations.yml
- Created the ID receiver transform
- Added an integration test for ID

### Testing
- [x] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [x] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Linked Issues
- Fixes #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?

### CODE REVIEW

File: prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/ID/ID-receiver-transform.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+
+constants:
+  hl7OBXField: '/PATIENT_RESULT/ORDER_OBSERVATION/OBSERVATION/OBX'
```
Comment (GilmoreA6) [inline prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/ID/ID-receiver-transform.yml:6]: this isn't used

File: prime-router/settings/STLTs/ID/id-phd.yml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        useTestProcessingMode: false
+        useBatchHeaders: true
+        receivingApplicationName: null
+        receivingApplicationOID: null
+        receivingFacilityName: null
+        receivingFacilityOID: null
+        messageProfileId: null
+        replaceValue: {}
+        replaceValueAwithB: {}
+        reportingFacilityName: "SimpleReport"
```
Comment (GilmoreA6) [inline prime-router/settings/STLTs/ID/id-phd.yml:25]: these aren't used in the UP

**Review by GilmoreA6 — APPROVED** [review]
only comment is a NIT

File: prime-router/settings/STLTs/ID/id-phd.yml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "id-phd"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation:
+        schemaName: "classpath:/metadata/hl7_mapping/receivers/STLTs/ID/ID-receiver-transform.yml"
```
Comment (oslynn) [inline prime-router/settings/STLTs/ID/id-phd.yml:15]: Please use "azure:/" instead of "classpath:/"

**Review by oslynn — APPROVED** [review]

**Pull request merged**